src/frontend/apps/standalone_site/src/features/Profile/components/ProfilePage.tsx

```diff
@@ -50,34 +43,20 @@ export const ProfilePage = () => {
   return (
     <Box>
       <Heading level={1}>{intl.formatMessage(messages.header)}</Heading>
-      <Box
-        background="#daeeff"
-        margin={{ vertical: 'small' }}
-        pad="medium"
-        round="8px"
-      >
-        <Box
-          background="white"
-          direction="column"
-          pad="medium"
-          round="8px"
-          elevation="even"
-        >
-          <Box
-            direction={breakpoint === 'large' ? 'row' : 'column'}
-            gap="medium"
-          >
-            <Field
-              value={userWithData?.full_name || null}
-              defaultMessage="No name provided"
-            />
-            <Field
-              value={userWithData?.email || null}
-              defaultMessage="No email provided"
-            />
-          </Box>
-        </Box>
-      </Box>
+      <WhiteCard gap="small">
+        <Input
+          value={userWithData?.full_name || intl.formatMessage(messages.noName)}
+          disabled
+          icon={<span className="material-icons">person</span>}
+          label={intl.formatMessage(messages.inputLabelName)}
```

Comment:
It is weird to have an input here, and the hover border seems strange.
Also the two inputs are now stacked vertically.
![image](https://github.com/openfun/marsha/assets/720491/902240fd-5829-44a0-9e07-20cc7b4712f1)

In staging we have this
![image](https://github.com/openfun/marsha/assets/720491/524e4b58-de5a-48ed-a189-c48ea42ead03)

Reply:
At the base I wanted to have something a bit more standard, with a label to understand what is what, but there is not such things in Cunningham. We can see sometimes input in read only to display informations so I went in this direction.

What about that? It is the cunningham input with read-only that I retouched to not get any input references neither hover or border?
![image](https://github.com/openfun/marsha/assets/25994652/c0c3f18e-4ff5-4de8-b8f3-dabee956bff8)

Reply:
https://github.com/openfun/marsha/pull/2479/commits/4bbe218a02bdfffd663aa223ca75b158d51d9f46